_posts/2022-01-17-release-1.14.3.md

```diff
@@ -0,0 +1,413 @@
+---
+layout: post
+title:  "Apache Flink 1.14.3 Release Announcement"
+date: 2022-01-17T08:00:00.000Z
+categories: news
+authors:
+- tweise:
+  name: "Thomas Weise"
+- MartijnVisser:
+  name: "Martijn Visser"
+  twitter: "martijnvisser82"
+
+The Apache Flink community released the second bugfix version of the Apache Flink 1.14 series.
+
+---
+
+The Apache Flink community released the second bugfix version of the Apache Flink 1.14 series.
+The first bugfix release was 1.14.2, being an emergency release due to an Apache Log4j Zero Day (CVE-2021-44228). Flink 1.14.1 was abandoned. 
+That means that this Flink release is the first bugfix release of the Flink 1.14 series which contains bugfixes not related to the mentioned CVE.
+
+This release includes 164 fixes and minor improvements for Flink 1.14.0. The list below includes bugfixes and improvements. For a complete list of all changes see:
+[JIRA](https://issues.apache.org/jira/secure/ReleaseNote.jspa?version=12351075&projectId=12315522).
+
+We highly recommend all users to upgrade to Flink 1.14.3.
+
+Updated Maven dependencies:
+
+```xml
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-java</artifactId>
+  <version>1.14.3</version>
+</dependency>
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-streaming-java_2.11</artifactId>
+  <version>1.14.3</version>
+</dependency>
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-clients_2.11</artifactId>
+  <version>1.14.3</version>
+</dependency>
+```
+
+You can find the binaries on the updated [Downloads page]({{ site.baseurl }}/downloads.html).
+
+
+
+
+
+        Release Notes - Flink - Version 1.14.3
```

Comment:
Can we remove test instabilities from this list?

Reply:
Done